blaze-server/src/test/scala/org/http4s/server/blaze/BlazeServerSuite.scala

```diff
@@ -119,11 +119,11 @@ class BlazeServerSuite extends Http4sSuite {
     }
 
   blazeServer.test("route requests on the service executor") { server =>
-    get(server, "/thread/routing").map(_.startsWith("io-compute-")).assertEquals(true)
+    get(server, "/thread/routing").map(_.startsWith("http4s-spec-")).assertEquals(true)
```

Comment:
I think that these were testing that they ran on a separate executor, and we might wish to make that more explicit by providing one for these tests. This feels like a test that shouldn't have had to change.

Reply:
I was also wondering if those changes have revealed an issue or not.
But I am not capable of answering that question myself.

Reply:
I don't think you've revealed an issue with the code, but I think with how the test is written. https://github.com/http4s/http4s/issues/4166